spec/spec_helper.rb

```diff
@@ -1,5 +1,5 @@
-# use local 'lib' dir in include path
-$:.unshift File.dirname(__FILE__)+'/../lib'
+# use . in include path
```

Comment:
it means this line will work:
`Dir[('spec/support/**/*.rb')].sort.each { |file| require file }`